.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: npm
+    directory: /web
+    schedule:
+      interval: weekly
+    assignees:
+      - akarachen
```

Comment:
`assignees` can be removed, as it's optional and inappropriate to be set with single user's name.